topics/perovskite/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: perovskite
+created_by: Paramesh Chandra
+display_name: Perovskite
+short_description: A perovskite is any material with a crystal structure similar to the mineral called perovskite, which consists of calcium titanium oxide (CaTiO3).
```

Comment:
Instead of providing the definition here, could you clarify what repositories in this topic are actually doing and why they exist? Maybe something like this:

> Material science research related to the mineral perovskite, which may be used to...

Reply:
I have changed the short description accordingly.